components/dashboard/src/whatsnew/MigrationPage.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { useMutation } from "@tanstack/react-query";
+import { OnboardingStep } from "../onboarding/OnboardingStep";
+import { User } from "@gitpod/gitpod-protocol";
+import { getGitpodService } from "../service/service";
+import { useOrganizationsInvalidator } from "../data/organizations/orgs-query";
+import { Separator } from "../components/Separator";
+import gitpodIcon from "../icons/gitpod.svg";
+import { useContext } from "react";
+import { UserContext } from "../user-context";
+
+export function MigrationPage() {
+    const migrateUsers = useMigrateUserMutation();
+
+    return (
+        <div className="container">
+            <div className="app-container">
+                <div className="flex items-center justify-center py-3">
+                    <img src={gitpodIcon} className="h-6" alt="Gitpod's logo" />
+                </div>
+                <Separator />
+                <div className="mt-24">
+                    <OnboardingStep
+                        title="Gitpod's Org Restructure"
+                        subtitle="Simplifying and Clarifying Your Gitpod Experience"
+                        isValid={true}
+                        isSaving={migrateUsers.isLoading}
+                        onSubmit={migrateUsers.mutateAsync}
+                    >
+                        <p className="text-gray-500 text-base">
+                            Gitpod is introducing a new approach to organizations, billing, and cost attribution.
+                            Organizations are now defined as a company, and all resources are fully owned by the
+                            organization. Personal accounts have been deprecated and automatically turned into an
+                            organization. Billing has been moved to the org settings, and cost attribution is now
+                            attributed to the org that owns the resource. For more information, check out our blog post:
+                            "A Guide to Gitpod's New Approach to Organizations, and Billing."
+                        </p>
+                    </OnboardingStep>
```

Comment:
Thanks! This step is not optional or would allow users any choice. So it doesn't make sense to display the menu on the top. The change has not a lot to do with collaboration so I think the headline is missleading.

Reply:
Looks shippable! 😃